tests/test_uta_parsers_seqrecord.py

```diff
@@ -71,29 +104,31 @@ def test_seq_record_facade(
         file_name,
         expected_id,
         expected_gene_symbol,
+        expected_gene_synonyms,
+        expected_gene_type,
         expected_gene_id,
+        expected_db_xrefs,
         expected_cds_se_i,
+        expected_cds_product,
+        expected_cds_protein_id,
+        expected_cds_translation,
         expected_exons_se_i,
     ):
         gbff_file = os.path.join(self.test_data_dir, file_name)
         seq_record = [sr for sr in SeqIO.parse(gbff_file, 'gb')][0]
         self.seq_record_facade = SeqRecordFacade(seq_record)
         assert self.seq_record_facade.id == expected_id
         assert self.seq_record_facade.gene_symbol == expected_gene_symbol
+        assert self.seq_record_facade.gene_synonyms == expected_gene_synonyms
+        assert self.seq_record_facade.gene_type == expected_gene_type
         assert self.seq_record_facade.gene_id == expected_gene_id
+        assert self.seq_record_facade.db_xrefs == expected_db_xrefs
         assert self.seq_record_facade.cds_se_i == expected_cds_se_i
+        assert self.seq_record_facade.cds_product == expected_cds_product
+        assert self.seq_record_facade.cds_protein_id == expected_cds_protein_id
+        assert self.seq_record_facade.cds_translation == expected_cds_translation
         assert self.seq_record_facade.exons_se_i == expected_exons_se_i
 
-    @parameterized.expand([
-        param('no genes', features={}),
-        param('no genes', features={'gene': []}),
-        param('more than one gene', features={'gene': [Mock(), Mock()]}),
-        param('more than one CDS', features={'CDS': [Mock(), Mock()]}),
-    ])
-    def test_validate_features_by_type_invalid(self, test_name, features):
```

Comment:
Are there tests that capture what was tested here?

Reply:
Let me look at that again to see if they are covered. If not, I can add them back.

Reply:
Thanks @nvta1209, I have added back these feature count validations